### PR TITLE
Add specification of device, manifests, tokens

### PR DESCRIPTION
**Pull request by alerei**

### CODE REVIEW

File: api/v2/device/license/activate/openapi.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "New Spec - OpenAPI 3.0",
+    "description": "This is a sample description about this spec.",
```
Comment (sulksamino) [inline api/v2/device/license/activate/openapi.json:5]: Was the placeholder title and description intentional?

File: api/v2/device/license/validate/openapi.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "New Spec - OpenAPI 3.0",
+    "description": "This is a sample description about this spec.",
```
Comment (sulksamino) [inline api/v2/device/license/validate/openapi.json:5]: Was the placeholder title and description intentional?

File: api/v2/manifests/openapi.json [hunk 3]
```diff
@@ -0,0 +1,184 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "New Spec - OpenAPI 3.0",
+    "description": "This is a sample description about this spec.",
```
Comment (sulksamino) [inline api/v2/manifests/openapi.json:5]: Was the placeholder title and description intentional?

File: api/v2/tokens/openapi.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "New Spec - OpenAPI 3.0",
+    "description": "This is a sample description about this spec.",
```
Comment (sulksamino) [inline api/v2/tokens/openapi.json:5]: Was the placeholder title and description intentional?

File: api/v2/device/license/activate/openapi.json [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+                    "reason": {
+                      "type": "string"
+                    },
+                    "statusCode": {
+                      "type": "integer"
+                    },
+                    "statusText": {
+                      "type": "string"
+                    }
+                  },
```
Comment (sulksamino) [inline api/v2/device/license/activate/openapi.json]:
```suggestion
                  }
```

**Review by sulksamino — CHANGES_REQUESTED** [review]

File: api/v2/device/license/activate/openapi.json (hunk 1 above)
Comment (alerei) [reply api/v2/device/license/activate/openapi.json:5]: To be honest, the files are just a copy of what is already there. Don't think it's useful to fix these new files only, but to restructure the specification in general:

https://flecs-technologies.atlassian.net/browse/FLX-563

**Pull request merged**